source/CardGames.py

```diff
@@ -190,11 +190,11 @@ def __init__(self):
         #Initialize the objects that need to be initialized
```

Comment:
Everything looks great! Is the functionality still all the same as before or has something changed?

Reply:
Generally the functionality is the same. Just some visual changes and a redo of some functions to prep them for future functions and turns.